src/vl-wizard.js

```diff
@@ -0,0 +1,120 @@
+import { VlElement, define } from '/node_modules/vl-ui-core/dist/vl-core.js';
+import '/node_modules/vl-ui-progress-bar/dist/vl-progress-bar-all.js';
+import '/lib/wizard.js';
```

Comment:
De diff tussen de originele wizard.js en degene in de lib folder:

```
3887c3887
<     var setFocus = arguments.length > 1 && arguments[1] !== undefined ? arguments[1] : true;
---
>     var setFocus = arguments.length > 1 && (arguments[1] !== undefined ? arguments[1] : true);
3917c3917
<   var _createWizard = function _createWizard(wizard, params) {
---
>   var _createWizard = function _createWizard(wizard, params, paneElements, stepElements, progressbarElement) {
3920c3920
<     var panes = wizard.querySelectorAll(".".concat(wizPaneClass));
---
>     var panes = paneElements || wizard.querySelectorAll(".".concat(wizPaneClass));
3923,3926c3923,3926
<     var ctasNext = wizard.querySelectorAll('[' + wizNextAtt + ']');
<     var ctasPrev = wizard.querySelectorAll('[' + wizPrevAtt + ']');
<     var pbSteps = wizard.querySelectorAll('[' + wizStepsAtt + ']');
<     var progressbarSelector = wizard.getElementsByClassName(pbClass)[0];
---
>     var ctasNext = [... panes].map((pane) => pane.querySelector('[' + wizNextAtt + ']')).filter(Boolean);
>     var ctasPrev = [... panes].map((pane) => pane.querySelector('[' + wizPrevAtt + ']')).filter(Boolean);
>     var pbSteps = stepElements || wizard.querySelectorAll('[' + wizStepsAtt + ']');
>     var progressbarSelector = progressbarElement || wizard.getElementsByClassName(pbClass)[0];
3931c3931
<     var flkty = new js(paneWrapper, {
---
>     var flkty = new js(panes[0].parentElement, {
3940c3940
<     var flkViewport = wizard.querySelector('.flickity-viewport');
---
>     var flkViewport = panes[0].closest('.flickity-viewport');
3946c3946
<           _setAdaptiveHeight(wizard, flkViewport);
---
>           _setAdaptiveHeight(panes[0].parentElement, flkViewport);
4013c4013
<       _setAdaptiveHeight(wizard, flkViewport);
---
>       _setAdaptiveHeight(panes[0].parentElement, flkViewport);
4027c4027
<       _setAdaptiveHeight(wizard, flkViewport);
---
>       _setAdaptiveHeight(panes[0].parentElement, flkViewport);
4030c4030
<     _setAdaptiveHeight(wizard, flkViewport);
---
>     _setAdaptiveHeight(panes[0].parentElement, flkViewport);
4047,4048c4047,4048
<       value: function dress(element, params) {
<         _createWizard(element, params);
---
>       value: function dress(element, params, panes, steps, progressbar) {
>         _createWizard(element, params, panes, steps, progressbar);
```

Reply:
Er waren inderdaad wat wijzigingen nodig, ik heb #3 voorzien.